src/components/InternshipEngagementListItemCard/InternshipEngagementListItem.graphql

```diff
@@ -32,4 +32,13 @@ fragment InternshipEngagementListItem on InternshipEngagement {
   completeDate {
     value
   }
+  project {
```

Comment:
Let's also add the `...Id `. It's good practice to always pull ids of object since that is what Apollo uses for caching.